indy_common/config.py

```diff
@@ -113,3 +113,5 @@
 
 CONTEXT_SIZE_LIMIT = 131072
 JSON_LD_LIMIT = CONTEXT_SIZE_LIMIT
+
+enableRichSchemas = False
```

Comment:
Suggest changing `enableRichSchemas` to `ENABLE_RICH_SCHEMAS` because it's some kind of constants. But in fact, looks like we don't have rules for config naming. So it's just my point of view and not necessary to change.